docs/docs/commands.md

```diff
@@ -89,37 +92,38 @@ This command sends a message of the current suggestions channel. If no suggestio
 
 ### Help Command
 ```
-,help
+,help [command]
 ```
-The help command displays all commands (commands will display based on permissions of the user) and additonal information dislaying current prefix, suggestions channel, bug reports links and the link to this documentation.
+The help command displays all commands (commands will display based on permissions of the user) and additonal information: current prefix, suggestions channel, a link to the website, a link to the GitHub page, and a link to the support server.  
+If you type a command aliase after `help`, the bot will display help information on the relevant command.
 
 Aliases: `h, halp`
 
 ### Info Command
 ```
 ,info
 ```
-The info command displays the bot author, bot description, the link to this documenation and the [Support Discord](https://discord.gg/ntXkRan). Some of the most important information is found here.
+The info command displays the bot author, bot description, links to the website, the [Support Discord](https://discord.gg/ntXkRan) and the Github page, and the bot version. Some of the most important information is found here.
 
 Aliases: `botinfo`
 
 ### Invite Command
 ```
 ,invite
 ```
-The invite command sends the user a DM with the bot invite link as well as the link to this documenation and the [Support Discord](https://discord.gg/ntXkRan).
+The invite command sends the user a DM with the bot invite link as well aslinks to the bot website, the [Support Discord](https://discord.gg/ntXkRan), and the GitHub page.
```

Comment:
I would add a hyperlink here to the [GitHub](https://github.com/suggestionsbot/suggestions-bot): `[GitHub](https://github.com/suggestionsbot/suggestions-bot)`